specs/vep-554-apache-airflow-integration/README.MD

```diff
@@ -395,6 +402,31 @@ Secrets will be stored as part of the Airflow Connection like follows:
 | connection.login    | Client ID     |
 | connection.password | Client Secret |
 
+For the connection itself, a standard Airflow Connection object would be used,
+
+```python
+Connection(
+  conn_id="",
+  conn_type="vdk",
+  description=None,
+  login="<client-id-or-empty>",
+  password="<password-or-oauth-token>",
```

Comment:
user(login) and password are not the same as client id / client secretor oauth token.
In the future we may support some type of Basic Authentication with user and password in which case those field would be used. 

please look at what other similar hooks/connectoins that support oauth2 do - https://github.com/airflow-plugins/marketo_plugin/blob/master/hooks/marketo_hook.py, https://github.com/airflow-plugins/google_sheets_plugin/blob/master/hooks/google_hook.py 

I suggest we put the required security configuration in extras.

Reply:
I don't think that's a concern. It will also mask extras if they contain certain keys
See https://airflow.apache.org/docs/apache-airflow/stable/security/secrets/mask-sensitive-values.html#sensitive-field-names

```
When masking is enabled, Airflow will always mask the password field of every Connection that is accessed by a task.
It will also mask the value of a Variable, or the field of a Connection's extra JSON blob 
if the name contains any words in ('access_token', 'api_key', 'apikey','authorization', 'passphrase', 'passwd', 'password', 'private_key', 'secret', 'token').
```

Reply:
But it is a core configuration, right?

Reply:
For example, the Databrick provider uses a password field for client_sercret.

Reply:
I see no problem in reusing the password field, but I am ok with both approaches.

Reply:
Updated the spec.